src/views/dashboard/graphs/Card.vue

```diff
@@ -48,18 +48,45 @@ limitations under the License. -->
         showUnit: true,
         textAlign: "center",
         metricConfig: [],
-        decorations: {},
+        valueMappings: {},
       }),
     },
   });
   const { t } = useI18n();
   const metricConfig = computed(() => props.config.metricConfig || []);
-  const decorations = computed(() => props.config.decorations || {});
+  const valueMappings = computed(() => props.config.valueMappings || {});
   const key = computed(() => Object.keys(props.data)[0]);
   const singleVal = computed(() =>
     Array.isArray(props.data[key.value]) ? props.data[key.value][0] : props.data[key.value],
   );
   const unit = computed(() => metricConfig.value[0] && encodeURIComponent(metricConfig.value[0].unit || ""));
+
+  function getValue() {
+    if (valueMappings.value[singleVal.value]) {
+      return valueMappings.value[singleVal.value];
+    }
+    const regex = /-?\d+(\.\d+)?/g;
+    const list = Object.keys(valueMappings.value);
+    for (const i of list) {
+      const k = i.replace(/\s+/g, "");
+      let withinRange = false;
+      const ranges = k.match(regex)?.map(Number) || [];
+      if (k.startsWith("[")) {
+        withinRange = k.startsWith("[-∞") || Number(singleVal.value) >= ranges[0];
+      } else {
+        withinRange = k.startsWith("(-∞") || Number(singleVal.value) > ranges[0];
+      }
+      if (k.endsWith("]")) {
+        withinRange = withinRange && (k.endsWith("+∞]") || Number(singleVal.value) <= (ranges[1] || ranges[0]));
```

Comment:
So, `+∞` and `-∞` are supported if we need no upper or lower boundary?

Reply:
Yep.

Reply:
I have a little concern of `∞`, it is hard to input from the keyboard. How about `infinite+` and `infinite-`

Reply:
If this supports writing regular expressions directly, how about that. For example, ^(10|[0-9])$ matches 0-10.

Reply:
`\b(2[5-9]|3[0-4])\b` could catch up, regex may not be as intuitive as the current method.

Reply:
@Fine0830 Could you try to change the format to include a regex?